packages/core/useFullscreen/index.ts

```diff
@@ -76,6 +76,15 @@ export function useFullscreen(
     ].find(m => (document && m in document) || (targetRef.value && m in targetRef.value)) as any
   })
 
+  const fullscreenElement = computed<'fullscreenElement' | undefined>(() => {
+    return [
+      'fullscreenElement',
+      'webkitFullscreenElement',
+      'mozFullScreenElement',
+      'msFullscreenElement',
+    ].find(m => (document && m in document)) as any
+  })
```

Comment:
Given this only checks for document, there's no need for this to be computed, as document is not going to be reactive